### PR TITLE
6231-removed-hardcoded-credentials-and-fixed-connecting

### PR DESCRIPTION
**Pull request by WiktoriaPrusik**

### CODE REVIEW

**Comment by WiktoriaPrusik** [comment]
@pwitusik can you take a look?

File: G1ANT.Addon.Ocr.Google/G1ANT.Addon.Ocr.Google/Api/GoogleCloudApi.cs [hunk 1]
```diff
@@ -7,49 +7,66 @@
...
             }
         }
+        private static GoogleCredential googleCredential;
 
-        private GoogleCredential CreateCredential()
+
+        public GoogleCloudApi()
+        {
+            if (googleCredential == null)
+                throw new Exception("trzeba sie zaogowac.");
```
Comment (pwitusik) [inline G1ANT.Addon.Ocr.Google/G1ANT.Addon.Ocr.Google/Api/GoogleCloudApi.cs]: Change to correct english message

Comment (WiktoriaPrusik) [reply G1ANT.Addon.Ocr.Google/G1ANT.Addon.Ocr.Google/Api/GoogleCloudApi.cs]: ups!!!

**Comment by lfronczyk** [comment]
I've got some issues (as usual) :P
When you use credential container you have to put there credential data. Where is it? How to do it? Where are the directions how to do that? How to maintain it across computers (in a near future a build machine will be one or more agents in the cloud)?
CC @pwitusik @danieladamkaiser

**Pull request merged**